Marlin/ultralcd.cpp

```diff
@@ -4710,7 +4725,7 @@ void lcd_update() {
     uint16_t bbr2 = planner.block_buffer_runtime() >> 1;
 
     #if ENABLED(DOGLCD)
-      if ((lcdDrawUpdate || drawing_screen) && (!bbr2 || (bbr2 > max_display_update_time)))
+      if ((lcdDrawUpdate || drawing_screen) && (!bbr2 || (bbr2 > max_display_update_time) || (currentScreen == lcd_sdcard_menu)))
```

Comment:
Is this only needed to support the feature, or does it fix an issue?

Reply:
It works around the Race Condition issue.   I have not found the root cause of the Race Condition yet.